README.md

```diff
@@ -87,18 +87,17 @@ For users of Intellij or comparable IDEs, we also suggest loading the checkstyle
 
 ### Dockstore Command Line
 
-The dockstore command line should be installed in a location in your path.
+Be sure to have a configuration file in place. Follow the instructions in [Configuration File](#configuration-file) for help.
 
-  /dockstore-client/bin/dockstore
+Add the directory `dockstore-client/bin` to your PATH: 
```

Comment:
I would say "Optionally add..." -- if I want to test a user's experience with the CLI, I'd want to use the official script, not a developer version.